crates/ruff_db/src/file_system.rs

```diff
@@ -7,9 +7,11 @@ use filetime::FileTime;
 
 pub use memory::MemoryFileSystem;
 pub use os::OsFileSystem;
+pub use vendored::VendoredFileSystem;
```

Comment:
I suggest moving `vendored` into its own module, considering that it isn't implementing `FileSystem`.

Reply:
Yes, I'm going to make that change. It still implements `FileSystem` in this PR currently, but I'm working on changing that.